src/argilla/client/feedback/integrations/huggingface/dataset.py

```diff
@@ -431,6 +466,11 @@ def from_huggingface(
                     value = hfds[index][f"{question.name}-suggestion"]
                     if question.type == QuestionTypes.ranking:
                         value = [{"rank": r, "value": v} for r, v in zip(value["rank"], value["value"])]
+                    elif question.type == QuestionTypes.span:
+                        value = [
+                            {"start": s, "end": e, "label": l}
+                            for s, e, l in zip(value["start"], value["end"], value["label"])
+                        ]
```

Comment:
perhaps we should add else statements here to be sure it raises errors/has behaviour when we add question types?

Reply:
I'm using the current approach to support span values mappings. I wouldn't add cross-cutting solution in the current version of the SDK. This is something that we can add for new SDK implementation